modules/miner/miner.go

```diff
@@ -180,8 +184,20 @@ func New(cs modules.ConsensusSet, tpool modules.TransactionPool, w modules.Walle
 		return nil, errors.New("miner subscription failed: " + err.Error())
 	}
 
+	// Automatically unsubscribe the miner from its transaction pool
```

Comment:
I feel like there should be no newline here because the unsubscribing of the consensus set is directly paired to the subscribing to the consensus set.